includes/admin/class-wcmypa-admin.php

```diff
@@ -459,6 +459,19 @@ class="wcmp__box wcmp__modal wcmp__shipment-summary__list"
         echo "</div>";
     }
 
+    /**
+     * @param  \WC_Order $order
+     *
+     * @return bool
+     */
+    public static function hasLocalPickup(WC_Order $order): bool
+    {
+        $shippingMethods      = $order->get_shipping_methods();
+        $shippingMethodId     = reset($shippingMethods)->get_method_id();
+
+        return WCMP_Shipping_Methods::LOCAL_PICKUP === $shippingMethodId;
```

Comment:
We moeten stoppen met WCMYPA vol zetten met onnodige methods (en überhaupt de hele plugin met static methods). 

Deze functie returnt iets over een WC_Order, ik zou hem dan ook op een plek zetten waar het normaal is om info over een order op te vragen, of in WCMP_Shipping_Methods oid